include/openPMD/Datatype.hpp

```diff
@@ -71,8 +71,6 @@ enum class Datatype : int
 
     BOOL,
 
-    DATATYPE = HIGHEST_DATATYPE,
```

Comment:
Let's check if `HIGHEST_DATATYPE` is used somewhere

Reply:
It's not